pkgs/development/libraries/opencascade-occt/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, fetchurl
+, cmake
+, tcl
+, tk
+, mesa_glu
+, libXext
+, libXmu
+, libXi
+, doxygen
+}:
+
+let version = "7.3.0";
+    patch = "p2";
+
+in stdenv.mkDerivation {
+
+  name = "opencascade-${version}${patch}";
+
+  src = fetchurl {
+    name = "occt-2a8846f.tar.gz";
+    url = "https://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=2a8846f92e43a12ed86b566de289c45eb3842d75;sf=tgz";
```

Comment:
Is there any better way to get the srcs from gitweb? As it is, you have to change 5 lines for each release.

Reply:
Reduced it down to four… There is also option to simply use the tag for the version, it's only two lines to change each time, but the version would look less standard.